### PR TITLE
Fix massive lags when EEC is running at high voltage

### PR DESCRIPTION
**Pull request by kuba6000**

### CODE REVIEW

**Review by miozune — COMMENTED** [review]
Actually `GT_MetaTileEntity_Hatch_OutputBus#storeAll`, which is called by `GT_MetaTileEntity_MultiBlockBase#addOutput`, handles splitting stacks

**Comment by kuba6000** [comment]
> Actually `GT_MetaTileEntity_Hatch_OutputBus#storeAll`, which is called by `GT_MetaTileEntity_MultiBlockBase#addOutput`, handles splitting stacks

It doen't split non stackable items
image

**Comment by miozune** [comment]
oh, bug in GT...

**Comment by kuba6000** [comment]
> oh, bug in GT...

fixed

File: src/main/java/kubatech/api/implementations/KubaTechGTMultiBlockBase.java [hunk 1]
```diff
@@ -168,9 +168,6 @@ protected int calculateOverclock(long aEUt, int aDuration, final long maxInputEU
             final ArrayList<ItemStack> stacks = new ArrayList<>(Arrays.asList(this.mOutputItems));
```
Comment (miozune) [inline src/main/java/kubatech/api/implementations/KubaTechGTMultiBlockBase.java]: Now this can be removed

**Review by miozune — APPROVED** [review]
This code probably can be migrated to ProcessingLogic, but after stable

**Pull request merged**